code/backend/Admin.py

```diff
@@ -2,12 +2,10 @@
 import os
 import logging
 import sys
-from dotenv import load_dotenv
 from azure.monitor.opentelemetry import configure_azure_monitor
+from batch.utilities.helpers.EnvHelper import EnvHelper
 
-
-load_dotenv()
-
+env_helper: EnvHelper = EnvHelper()
```

Comment:
Why was it added, it doesn't look like it's used anywhere?

Also, we should import and use the env helper after we've set up application insights below, otherwise it doesn't work correctly